src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+@use './blocks/card.scss';
+@use './blocks/stars.scss';
+
+* {
```

Comment:
Don't use * selector (it impacts performance).
You can add these properties to the ``html`` or ``body``.

![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/a94b94d3-15ab-4d8d-98bf-88f461ec7264)